lib/src/main/java/com/github/featuredetectlib/FeatureDetector.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.featuredetectlib
+
+typealias Descriptor = List<Float>
+
+/**
+ * Detects keypoints and their descriptors on an image.
+ */
+interface FeatureDetector {
+    /**
+     * Width of the images to be processed.
+     */
+    var width: Int
+
+    /**
+     * Height of the passed to be processed.
```

Comment:
Maybe it would be better to clarify: "passed image"?

Reply:
Yep, that's a typo